### PR TITLE
Move TxMake process during spooling

### PR DESCRIPTION
**Pull request by jdent02**
Now it is done as part of the .alf queue

### CODE REVIEW

**Comment by bsavery** [comment]
The better way is to just use the -newer option on txmake.

**Comment by jdent02** [comment]
Done.

File: spool.py [hunk 1]
```diff
@@ -113,43 +117,52 @@ def spool_render(rman_version_short, rib_files, denoise_files, denoise_aov_files
...
-                    cmd_str = ['denoise'] + denoise_options + [f[0] for f in denoise_files[frame_num - 3: frame_num]] + files
+                files = [item for sublist in denoise_aov_files[frame_num - frame_begin -
+                                                               2: frame_num - frame_begin + 1] for item in sublist] if rm.spool_denoise_aov else []
+                cmd_str = ['denoise'] + denoise_options + [f[0] for f in denoise_files[
+                    frame_num - frame_begin - 2: frame_num - frame_begin + 1]] + files
                 write_cmd_task_line(f, 'Denoise frame %d' % (frame_num - 1),
                                     [('PixarRender', cmd_str)], 3)
             if frame_num == frame_end:
-                denoise_options.remove('-F 1 -L 1')
+                try:
```
Comment (bsavery) [inline spool.py]: Hey one more thing:  try/excepts shouldn't be used here, instead do:

if ('-F 1 -L 1') in denoise_options:
     denoise_options.remove(...)

**Comment by bsavery** [comment]
It looks like the commits from this and the other are combined?

**Comment by jdent02** [comment]
Crap.  Yeah I had merged a number of changes to master into the branch I use for development.  This branch doesn't reflect the change I made to my master patch yesterday.  I'll fix it.

**Pull request closed without merging**